src/domain/swaps/entities/__tests__/order.builder.ts

```diff
@@ -0,0 +1,101 @@
+import { Order } from '@/domain/swaps/entities/order.entity';
+import { Builder, IBuilder } from '@/__tests__/builder';
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+
+export function orderBuilder(): IBuilder<Order> {
+  return new Builder<Order>()
+    .with('sellToken', getAddress(faker.finance.ethereumAddress()))
+    .with('buyToken', getAddress(faker.finance.ethereumAddress()))
+    .with(
+      'receiver',
+      faker.datatype.boolean()
+        ? getAddress(faker.finance.ethereumAddress())
+        : null,
+    )
+    .with('sellAmount', faker.number.bigInt({ min: 1 }))
+    .with('buyAmount', faker.number.bigInt({ min: 1 }))
+    .with('validTo', faker.date.future().getTime())
+    .with('appData', faker.string.hexadecimal())
+    .with('feeAmount', faker.number.bigInt({ min: 1 }))
+    .with('kind', faker.helpers.arrayElement(['buy', 'sell']))
+    .with('partiallyFillable', faker.datatype.boolean())
+    .with(
+      'sellTokenBalance',
+      faker.helpers.arrayElement(['erc20', 'internal', 'external']),
+    )
+    .with('buyTokenBalance', faker.helpers.arrayElement(['erc20', 'internal']))
+    .with(
+      'signingScheme',
+      faker.helpers.arrayElement(['eip712', 'ethsign', 'presign', 'eip1271']),
+    )
+    .with('signature', faker.string.hexadecimal())
+    .with(
+      'from',
+      faker.datatype.boolean()
+        ? getAddress(faker.finance.ethereumAddress())
+        : null,
+    )
+    .with('quoteId', faker.datatype.boolean() ? faker.number.int() : null)
+    .with('creationDate', faker.date.recent())
+    .with('class', faker.helpers.arrayElement(['market', 'limit', 'liquidity']))
+    .with('owner', getAddress(faker.finance.ethereumAddress()))
+    .with('uid', faker.string.uuid())
+    .with(
+      'availableBalance',
+      faker.datatype.boolean() ? faker.number.bigInt({ min: 1 }) : null,
+    )
+    .with('executedSellAmount', faker.number.bigInt({ min: 1 }))
+    .with('executedSellAmountBeforeFees', faker.number.bigInt({ min: 1 }))
+    .with('executedBuyAmount', faker.number.bigInt({ min: 1 }))
+    .with('executedFeeAmount', faker.number.bigInt({ min: 1 }))
+    .with('invalidated', faker.datatype.boolean())
+    .with(
+      'status',
+      faker.helpers.arrayElement([
+        'presignaturePending',
+        'open',
+        'fulfilled',
+        'cancelled',
+        'expired',
+      ]),
+    )
+    .with('fullFeeAmount', faker.number.bigInt({ min: 1 }))
+    .with('isLiquidityOrder', faker.datatype.boolean())
+    .with(
+      'ethflowData',
+      faker.datatype.boolean()
+        ? {
+            refundTxHash: faker.string.hexadecimal(),
```

Comment:
As this is nullable in the schema, what about `refundTxHash: faker.datatype.boolean() ? faker.string.hexadecimal() : null`

Reply:
Changed in [e3a4a9c](https://github.com/safe-global/safe-client-gateway/pull/1299/commits/e3a4a9c33aac2d31572dd3cb51e6cacec9ed3515)